translator/gobgp.py

```diff
@@ -22,7 +22,11 @@ def _get_family(self, ip_version):
         else:
             return gobgp_pb2.Family.AFI_IP
 
-    def _build_path(self, ip):
+    def _build_path(self, ip, event_data):
+        logging.debug(f"ip: {ip}, event_data: {event_data}")
+        asn = event_data.get("asn", 64500)
+        community = event_data.get("community", 666)
```

Comment:
You should consider trying to cast `asn` and `community` explicitly as `int`s here, and maybe even try/except/raise for when a non-int gets passed through. Bonus points for a test that looks for that exception.

Reply:
Actually, I think an even better place to do that is in `translator.py` L24, where we actually first create that `event_data` structure. 

All of this is probably unrelated to the CI issues, and is overall minor, so feel very free to ignore!